packages/account/src/Styles/account.scss

```diff
@@ -643,6 +649,86 @@ $MIN_HEIGHT_FLOATING: calc(
             padding-left: 8px;
         }
     }
+    .login-history-container {
+        height: 100%;
+        width: 100%;
+
+        .login-history-table {
+            @include desktop {
+                height: 100%;
+            }
+
+            @include mobile {
+                width: 90%;
+                border-collapse: separate;
+                border-spacing: 0 1em;
+                margin: 10px auto;
+                padding: 10px;
+                color: var(--text-general);
+                background-color: $color-grey-4;
+
+                &.theme-dark {
+                    background-color: $color-black-3;
+                }
+                .cell-title {
+                    font-size: 12px;
+                    font-weight: bold;
+                    font-stretch: normal;
+                    font-style: normal;
+                    line-height: 1.5;
+                    letter-spacing: normal;
+                    color: var(--text-prominent);
+                }
+                .cell-value {
+                    font-size: 14px;
+                    font-weight: normal;
+                    font-stretch: normal;
+                    font-style: normal;
+                    line-height: 1.43;
+                    letter-spacing: normal;
+                }
+                .action {
+                    text-transform: capitalize;
+                }
+            }
+
+            &__table {
+                height: calc(100% - 42px);
+                flex: 1;
+            }
+            &__row {
+                /*             Date and Time Action Browser IP Status */
+                grid-template-columns: 210px 85px 185px 130px 90px;
```

Comment:
Why not use `fr` units?

Reply:
Using `fr` does not match the design. Tried many different combinations already :)